.gitignore

```diff
@@ -39,3 +39,4 @@ node_modules/
 .ruby-version
 /yarn-error.log
 /test-reports/
+tmp/cache/bootsnap/compile-cache-iseq 
```

Comment:
- [ ] [OPTIONAL] You may have ignored these big files but you still need to go the `feature/process_data_model` branch repo and also remove them also from the branch repo.